packages/website/content/integrations/examples/UseControllerIntegration.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { useForm, useController } from 'react-hook-form';
+import { TextInput, Form, FormControl } from '@contentful/f36-forms';
+import { Button } from '@contentful/f36-components';
+
+export default function ReactHookForm() {
```

Comment:
💅 Could you add please `Example` to the file name as well, just for the sake of consistency.
```suggestion
export default function UseControllerIntegrationExample() {
```